atomic_reactor/plugins/pre_stop_autorebuild_if_disabled.py

```diff
@@ -52,13 +52,16 @@ def _is_rebuild_enabled(self):
                               'enabled' if result else 'disabled',
                               self.config_file)
```

Comment:
If you initialize `result` to `False`, you only need to adjust the log messages and set it to `True` here.